### DISCUSSION TITLE
Add 12 and 24 hours refresh options

### DISCUSSION THREAD

**Discussion by sandboiii** [jocmp/capyreader#217] [Feature Requests]
Hey, I have only a few subscriptions, which rarely get any updates. So, I don't want to refresh them every hour and lose the battery charge. Is it possible to add more refresh options? I can make a pr if you let

**Comment by jocmp** [comment]
12 and 24 hour options have been added with version [2024.08.1030](https://github.com/jocmp/capyreader/releases/tag/2024.08.1030).

**Comment by sandboiii** [comment]
Thank you! It looks great!
